deploy/manifests/dev/us-east-2/tenant/storetheindex/heyfil/pvc.yaml

```diff
@@ -9,5 +9,5 @@ spec:
     - ReadWriteOnce
   resources:
     requests:
-      storage: 1Gi
+      storage: 2Gi
```

Comment:
This change is for the dev instance. checking that it's intended.

Reply:
yes - that's where we've promoted the recent change / are seeing disk full errors